packages/react-components/src/components/MessageThread.tsx

```diff
@@ -625,11 +631,13 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
     setIsAtTopOfScrollRef(atTop);
 
     // Make sure we do not stuck at the top if more messages are being fetched.
-    if (chatScrollDivRef.current.scrollTop === 0 && !isAllChatMessagesLoadedRef.current)
+    if (chatScrollDivRef.current && chatScrollDivRef.current.scrollTop === 0 && !isAllChatMessagesLoadedRef.current) {
```

Comment:
chatScrollDivRef.current?.scrollTop === 0
Will this work?

Reply:
Ohh, you might wanna pass the type guard in following line

Reply:
Oh I already have a 
```
if (!chatScrollDivRef.current) {
  return;
}
```
I can remove this extra check

Reply:
Removed